app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -23,7 +38,7 @@ def nics(extra_attributes = {})
       attributes = {
         :model_class => ::Network,
         :manager_ref => [:hardware, :ipaddress],
-        :association => :nics
+        :association => :nics,
```

Comment:
Can you move the associations for nics/disks/etc... that you currently added to the `Ovirt::InfraManager` up to the base `InfraManager` here?